src/main/java/io/stargate/sgv2/jsonapi/service/cqldriver/executor/NamespaceCache.java

```diff
@@ -42,6 +44,18 @@ protected Uni<CollectionSettings> getCollectionProperties(String collectionName)
           .transformToUni(
               (result, error) -> {
                 if (null != error) {
+                  // not a valid collection schema
+                  if (error instanceof JsonApiException
+                      && ((JsonApiException) error).getErrorCode()
+                          == ErrorCode.VECTORIZECONFIG_CHECK_FAIL) {
+                    return Uni.createFrom()
+                        .failure(
+                            new JsonApiException(
+                                ErrorCode.COLLECTION_NOT_EXIST,
```

Comment:
But wouldn't this give "Collection does not exist, collection name: " failure? Would it better to indicate that there is a table but one that is not valid Collection? Or maybe that happens somewhere else?